artemis-server/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/wireformat/BackupRequestMessage.java

```diff
@@ -34,12 +34,14 @@ public BackupRequestMessage() {
    }
 
    public BackupRequestMessage(int backupSize,
+                               SimpleString nodeID,
                                String journalDirectory,
                                String bindingsDirectory,
                                String largeMessagesDirectory,
                                String pagingDirectory) {
       super(BACKUP_REQUEST);
       this.backupSize = backupSize;
+      this.nodeID = nodeID;
```

Comment:
Please make a v2 version, this would be a breaking change, for anyone with existing clusters